NitroxModel/Logger/Log.cs

```diff
@@ -63,19 +63,24 @@ public static void Setup(bool asyncConsoleWriter = false, InGameLogger inGameLog
                              cnf.WriteTo.ColoredConsole(outputTemplate: consoleTemplate);
                          }
                      })
-                     .WriteTo.Logger(cnf => cnf
-                                            .Enrich.FromLogContext()
-                                            .WriteTo
+                     .WriteTo.Logger(cnf => {
+                        if (!useFileLogging) {
+                            return;
+                        }
+                        
+                         cnf.Enrich.FromLogContext()
+                                          .WriteTo
 #if DEBUG
-                                            .Map(nameof(PlayerName), "", (playerName, sinkCnf) => sinkCnf.Async(a => a.File(Path.Combine(LogDirectory, $"{GetLogFileName()}{playerName}-.log"),
+                                          .Map(nameof(PlayerName), "", (playerName, sinkCnf) => sinkCnf.Async(a => a.File(Path.Combine(LogDirectory, $"{GetLogFileName()}{playerName}-.log"),
 #else
                                             .Async((a => a.File(Path.Combine(LogDirectory, $"{GetLogFileName()}-.log"),
 #endif
-                                                                                                                            outputTemplate: "[{Timestamp:HH:mm:ss.fff}] [{Level:u3}{IsUnity}] {Message}{NewLine}{Exception}",
-                                                                                                                            rollingInterval: RollingInterval.Day,
-                                                                                                                            retainedFileCountLimit: 10,
-                                                                                                                            fileSizeLimitBytes: 200000000, // 200MB
-                                                                                                                            shared: true))))
+                                                                                                                          outputTemplate: "[{Timestamp:HH:mm:ss.fff}] [{Level:u3}{IsUnity}] {Message}{NewLine}{Exception}",
+                                                                                                                          rollingInterval: RollingInterval.Day,
+                                                                                                                          retainedFileCountLimit: 10,
+                                                                                                                          fileSizeLimitBytes: 200000000, // 200MB
+                                                                                                                          shared: true)));
+                     })
```

Comment:
1. The indentation does not conform the rest of the code.
2. The logger shouldn't be added if it needs to be skipped, as an optimization.

Reply:
About point 2: The code above it does the exact same `return;`-style way of toggling it. Will fix point 1 though.

Reply:
True, should also be changed. Here's a pastebin with new extension methods to optimize this code and reduce `Setup` method code size:
https://pastebin.com/MhGHQtcb

Reply:
Fixed in newest commits.